compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -5030,6 +5030,9 @@ private boolean isPresent(Node node) {
 
     private boolean checkIfAnonymous(Node node) {
         Node parent = node.parent();
+        if (parent.kind() == SyntaxKind.DISTINCT_TYPE_DESC) {
```

Comment:
Can we collapse this if statement with the next line's return statement?

Reply:
fixed